test/client/multi_percolate_test.rb

```diff
@@ -138,4 +138,4 @@ def populate!
     @index.refresh
   end
   # rubocop:enable Metrics/MethodLength
-end
+end unless $client.version_support.es_version_7_plus?
```

Comment:
While this method of conditionally not running the test will work, it doesn't let the file communicate to the test runner that there are tests that have been skipped. For example, running this test against ES 7 outputs the following:
```
0 runs, 0 assertions, 0 failures, 0 errors, 0 skips
```

Alternatively, if a `skip` is conditionally put into the `before` block of the outermost `describe`, then the test runner will know about the skipped tests and output the following:
```
3 runs, 0 assertions, 0 failures, 0 errors, 3 skips
```

Can you make that change here and in `percolator_test.rb`? I see there are other instances of that type of skip in this repo. To be consistent, the logic would be something like the following as the first few lines in the `before` block:

```
if $client.version_support.es_version_7_plus?
  skip "MultiPercolate not supported in ES version #{$client.version}"
end
```

Reply:
Got it, I'll make these changes. Thanks!

Reply:
Done!